extractor/filesystem/containers/k8simage/testdata/not-yaml.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: apps/v1
```

Comment:
Is there a reason to make this look very close like a yaml file? 
Would it be more obvious if this just does not follow the structure at all?

Right now this looks like a reasonable file that probably has some parsing error somewhere. But it is not immediately obvious (as in when you glance over the file) where that error comes from.